README.md

```diff
@@ -126,9 +126,9 @@ const transfer = {
     timeout: 0n, // u64, in nano-seconds.
     // Collection of accounts usually grouped by the currency: 
     // You can't transfer money between accounts with different ledgers:
-    ledger: 720,  // u32, ledger for transfer (e.g. currency).
+    code: 720,  // u32, ledger for transfer (e.g. currency).
     // Chart of accounts code describing the reason for the transfer:
-    code: 1,  // u16, (e.g. deposit, settlement)
+    ledger: 1,  // u16, (e.g. deposit, settlement)
```

Comment:
Just swap the values, since the comments apply to the field names as they are.

Reply:
Good catch, fixed. Thank you!